### PR TITLE
docs(www): Add hooks to static-query docs

### PR DESCRIPTION
**Pull request by KenzoBenzo**
## Description
Referenced from Jason Lengstorf's Gatsby + GraphCMS live-stream webinar.

### CODE REVIEW

**Comment by wardpeet** [comment]
on a side not should we mention `Gatsby version 2.1`, I would just omit it.

**Comment by marcysutton** [comment]
I agree with @DSchau that since there are already two docs, I don't think it makes sense to double up the detailed information. A link from the static-query doc to useStaticQuery would be very helpful, though!

In general, I do think the `Gatsby version 2.1` part is useful, at least for a little while, since useStaticQuery was released in a minor version. But it's already in the useStaticQuery doc, so leaving it out of a mention here would create less work in the future (only one place to update)!

File: docs/docs/static-query.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ export default () => (
 
 Using `StaticQuery`, you can colocate a component with its data. No longer is it required to, say, pass data down from `Layout` to `Header`.
 
+### Hooks Example
```
Comment (sidharthachatterjee) [inline docs/docs/static-query.md]:
```suggestion
### useStaticQuery
```

File: docs/docs/static-query.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ export default () => (
...
+
+Using the hooks version of static query is very similar. Checkout the documentation of `useStaticQuery` [here!](/docs/use-static-query/)
```
Comment (sidharthachatterjee) [inline docs/docs/static-query.md]:
```suggestion
There's also a hooks version of StaticQuery called `useStaticQuery`. Check out its documentation [here!](/docs/use-static-query/)
```

**Pull request merged**